### ISSUE TITLE
Be able to change the default SerializersModule

### ISSUE THREAD

**Issue by baillyjamy** [rjaros/kvision#230]
Hello,

I have a Get rest call that I can receive two types of data: `B` or `C`. 
`B` and `C` are children of `A`. `A` is an abstract class. There is no serialization / deserialization problem on the JVM. But with Kotlin/JS I need to register my subclasses in a `SerializersModule` like this : 
```kotlin
val module = SerializersModule {
    polymorphic(A::class) {
        subclass(B::class)
        subclass(C::class)
    }
}
```
Cf : https://github.com/Kotlin/kotlinx.serialization/issues/1077

After registering my module in a Json instance, I can easily deserialize my object in Kotlin/JS env like this : 
```Kotlin
val jsonString = "{"type":"B",...}"
val format = Json { serializersModule = module }
val instanceB: B = format.decodeFromString(jsonString)

```
So I would like to change default SerializersModule of an instance of RestClient.
Is it possible or can this feature be implemented?
Or do you have a better solution?

**Comment by rjaros** [comment]
For now there is no way to define custom serializer modules. You can of course just stick to dynamic/text and serialize/deserialize the data externally. 

But I think it should be fairly easy to add support for this in the next release by adding an optional parameter to the `RectClient` constructor.

**Comment by baillyjamy** [comment]
At the moment I just created a RestClient function extension with a paste copy of remoteCall() and add a parameter for the SerializerModule.
A little ugly but it will be temporary while waiting for the SerializerModule to be added in the builder.
By the way, it might be interesting to add SerializerModule as a parameter of the two remoteCall() that handle deserialization if we want to use a specific module just once. What do you think about it?

**Comment by rjaros** [comment]
It could be useful for input data serialization as well, so I think it would be easier to make it available for all methods.

**Comment by rjaros** [comment]
Released with 4.0.0
